int32 solvePnpType
---
geometry_msgs/TransformStamped rigidBodyTransform
